src/e2e_tests/sample_scripts/test_search.py

```diff
@@ -0,0 +1,36 @@
+from common.pages import Page
+import allure
+from common.pages import is_displayed
+from delayed_assert import expect, assert_expectations
+from allure import severity, severity_level
+
+search_request = "wor"
+
+
+@severity(severity_level.NORMAL)
+@allure.title("Check search results")
+def test_search_by_fixed_keyword_case_insensitive(browser, config_host, scripts):
+    home_page = Page(browser, config_host)
+    home_page.load()
+
+    home_page.sidebar_search_button.click()
+    home_page.sidebar_search_input.send_keys(search_request)
```

Comment:
Please replace the variable with the string literal, it makes the test easier to read